interface.go

```diff
@@ -85,7 +85,7 @@ type GlobalLookupFactory interface {
 	// Help text for the CLI
 	Help() string
 	// Return a single scanner which will scan a single host
-	MakeRoutineFactory() (RoutineLookupFactory, error)
+	MakeRoutineFactory(int) (RoutineLookupFactory, error)
```

Comment:
The head of master has 220 golint errors. This branch reduces the number of golint errors down to 150. I suggest we tackle conforming to golint as a separate PR across the entire project.